src/index.ts

```diff
@@ -61,4 +72,15 @@ export function activate() {
 const preProcess = (/** @type {string} */source) =>
     source
         .replace(/\</g, '&lt;')
-        .replace(/\>/g, '&gt;');
\ No newline at end of file
+        .replace(/\>/g, '&gt;');
+
+function injectMermaidTheme(md) {
+    const render = md.renderer.render;
+    md.renderer.render = function() {
+        return `<span id="${configSection}"
```

Comment:
Perhaps this should be a `<meta>` tag or at least set `aria-hidden` so it is not treated as part of the document for accessibility

Reply:
I haven't found how to modify `<head>` part from the renderer, so just set `aria-hidden`.